update.go

```diff
@@ -122,11 +122,11 @@ other options are ignored.
 		},
 		cli.StringFlag{
 			Name:  "memory-swap",
-			Usage: "Total memory usage (memory + swap); set '-1' to enable unlimited swap",
+			Usage: "Total memory usage (memory + swap); use '-1' to enable unlimited swap",
 		},
 		cli.IntFlag{
 			Name:  "pids-limit",
-			Usage: "Maximum number of pids allowed in the container",
+			Usage: "Maximum number of tasks; use '-1' for unlimited",
```

Comment:
Maybe some users will be confused with `-r` and `runtime-spec`?
```
runc update -r - containerid
{
  "pids": { "limit": 0 }
}
```

Reply:
1. This is the way it was always working.
2. It seems that the behavior of `runc update` is not in spec.